lib/l10n/intl_it.arb

```
@@ -751,5 +751,92 @@
   "@_comment_timeout_messages": "Messaggi di notifica timeout",
   "orderTimeoutTaker": "Non hai risposto in tempo. L'ordine sarà ripubblicato",
   "orderTimeoutMaker": "La tua controparte non ha risposto in tempo. L'ordine sarà ripubblicato",
-  "orderTimeout": "Timeout dell'ordine verificato"
+  "orderTimeout": "Timeout dell'ordine verificato",
+  
+  "@_comment_notifications": "Testo Schermata Notifiche",
+  "notifications": "Notifiche",
+  "noNotifications": "Nessuna notifica",
+  "noNotificationsDescription": "Non hai ancora notifiche. I nuovi avvisi di scambio e gli aggiornamenti appariranno qui.",
+  "errorLoadingNotifications": "Errore nel caricamento delle notifiche",
+  "retry": "Riprova",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)